.github/workflows/superset-python.yml

```diff
@@ -1,5 +1,5 @@
-# Python unit tests
-name: Python
+# Python Misc unit tests
+name: Python Misc
```

Comment:
Maybe rename the file to `superset-python-misc.yml`, too?

Reply:
done